pysb/bng.py

```diff
@@ -33,6 +33,10 @@
 except NameError:
     basestring = str
 
+try:
+    from collections.abc import Sequence
+except ImportError:
```

Comment:
We've [dropped Python 2.7 support](#414) recently, so there's no need for the `try`/`except` here. Simply importing from `collections.abc` should suffice. Applies to the other instances as well.

Reply:
Thanks, removed them. The `setup.py` still has the classifier Python 2 and hence I added these try/except block.

Reply:
Thanks. We're still in the process of removing Python 2 remnants from our codebase - thanks for bringing that one to my attention!